.github/workflows/test.yaml

```diff
@@ -25,3 +25,27 @@ jobs:
               run: yarn global add prettier@2.2.1
             - name: Check coding style
               run: ~/.yarn/bin/prettier --check assets --config .prettierrc.json
+
+    build-test:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
```

Comment:
I would suggest to use https://github.com/shivammathur/setup-php: it would avoid having a changing PHP version in tests and it would install extensions and tools faster